src/itp.bib

```diff
@@ -0,0 +1,120 @@
+@inproceedings{klein09,
+ author = {Klein, Gerwin and Elphinstone, Kevin and Heiser, Gernot and Andronick, June and Cock, David and Derrin, Philip and Elkaduwe, Dhammika and Engelhardt, Kai and Kolanski, Rafal and Norrish, Michael and Sewell, Thomas and Tuch, Harvey and Winwood, Simon},
+ title = {seL4: Formal Verification of an OS Kernel},
```

Comment:
acronyms in bibtex titles need to be in braces to avoid capitalization changes: `{OS}`